.github/workflows/posm-build.yml

```diff
@@ -0,0 +1,79 @@
+name: CI
+
+on:
+  push:
+    tags:
+      - posm-build-v*
+
+jobs:
+  build:
+    name: Run build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout react-store
+        uses: actions/checkout@v2
+        with:
+          repository: posm/react-store
+          ref: develop
+          path: src/vendor/react-store
+      - name: Checkout re-map
+        uses: actions/checkout@v2
+        with:
+          repository: posm/re-map
+          ref: develop
+          path: src/vendor/re-map
```

Comment:
Maybe we cannot use develop branch.
I have created tags `posm-replay-client-v1` from my local configuration on both re-map and react-store.